kobo/apps/superuser_stats/views.py

```diff
@@ -152,8 +156,8 @@ def domain_report(request):
         f'<code style="background: lightgray">start_date</code> parameter to <code style="background: lightgray">YYYY-MM-DD</code> and/or the '
         f'<code style="background: lightgray">end_date</code> parameter to <code style="background: lightgray">YYYY-MM-DD</code>.<br><br>'
         f'<b>Example:</b><br>'
-        f'<a href="{url}?start_date={today}&end_date={tomorrow}">'
-        f'  {url}?start_date={today}&end_date={tomorrow}'
+        f'<a href="{url}?start_date={today}&end_date={today}">'
+        f'  {url}?start_date={today}&end_date={today}'
```

Comment:
I used the today and tomorrow's dates in the examples to make sure it was clear which date should be the earlier one. Should we keep it that way? Otherwise it looks good to merge

Reply:
Right. As you've noticed I removed `tomorrow` everywhere. We were using `tomorrow` to be sure to include `today` inside the date range. There were an issue with that when filtering SQL queries with date objects and datetime fields. 


```python
today = '2022-11-25'
tomorrow = '2022-11-26'

# Working with datetime
Model.objects.filter(date_created__range=(today, today))  # return None
Model.objects.filter(date_created__range=(today, tomorrow))  # return every object created today

# Working with dates
Model.objects.filter(date_created__date__range=(today, today))  # return every object created today
Model.objects.filter(date_created__date__range=(today, tomorrow))  # return every object created today and tomorrow
```

It was a little bit confusing, especially when you only want to narrow down with the end date.
For example: User wants everything until '2022-11-10', they had to use `?end_date=2022-11-11` 

If you want, we can change the default value of `start_date` and update the example accordingly.  Also, write down explicitly the `end_date` is inclusive.